README.md

```diff
@@ -452,40 +464,84 @@ data:
   tokenurl: "<auth_url>"
   tokenurlsuffix: "/oauth/token"
 ```
+with TLS configurations:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sap-btp-service-operator-tls
+  namespace: <namespace>
+type: kubernetes.io/tls
+data:
+  tls.crt: <crt> #base64 encoded
+  tls.key: <key> #base64 encoded
+```
+
+#### Centrally-Managed Namespace Secret
+
+Define a secret named: `<namespace>-sap-btp-service-operator` in the Centrally Managed Namespace.
+
+**Note:**
+The system's centrally-managed namespace is set by the value in `.Values.manager.management_namespace`. You can provide this value during installation (refer to step 4 in the [Setup](#setup) section). 
+If you don't specify this value, the system will use the installation namespace as the default.
```

Comment:
add the name of the installation namespace

Reply:
it is given in the installation:
helm upgrade --install sap-btp-operator sap-btp-operator/sap-btp-operator \
    --create-namespace \
    --namespace=sap-btp-operator \